src/sections/case/skatteverket/index.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import styled from "styled-components";
+import { Container, Row, Col } from "react-bootstrap";
+
+import { Section, Box, Title, Text } from "../../../components/Core";
+
+import imgCase1 from "../../../assets/image/case/skatteverket.jpeg";
+import { marginLeft } from "styled-system";
+
+const ImgContainer = styled(Box)`
+  border-radius: 10px;
+  overflow: hidden;
+`;
+
+const CaseDetails = () => (
+  <>
+    <Section className="position-relative" borderBottom="1px solid #eae9f2;">
+      <Container>
+        <ImgContainer className="text-center" pb={["50px", null, "75px"]}>
+          <img src={imgCase1} alt="" className="img-fluid" />
+        </ImgContainer>
+
+        <Row>
+          <Col lg="12" xl="10" className="offset-xl-1">
+            <Box pb={["40px", null, "65px"]}>
+              <Title variant="card" mb="18px">
+                För en säkrare kvittohantering
+              </Title>
+              <Text variant="small">
+                Tillsammans med Skatteverket och Karios Future har vi utvecklat
+                ett självreglerande system för hantering av digitala kvitton.
+                Lösningen är en prototyp av en teknisk arkitektur med syfte att
+                säkerställa kvitton och fakturor i bokföringen och ge dessa en
+                unik identitet. Det försvårar misstag och fusk och kan bidra
+                till att reducera momsbedrägerier. Momsbedrägerier är ett
+                omfattande problem och EU-kommissionen uppskattar
+                momsbedrägerier inom EU till 137 miljarder Euro per år. Den
+                digitala lösningen möjliggör även automatkontering vilket
+                underlättar kvittohantering för affärer, slutkunder och
+                redovisningsaktörer.
+              </Text>
+            </Box>
+            <Box pb={["40px", null, "65px"]}>
+              <Title variant="card" mb="18px">
+                Demonstration av tekniken
+              </Title>
+              <Text variant="small">
+                Konceptet bygger på att digitala kvitton omvandlas hashar,
+                vilket är oigenkännliga textsträngar. Genom digital signaturer
+                kan kvittornas legitimitet sedan verifieras. På detta sätt
```

Comment:
```suggestion
                kan kvittonas legitimitet sedan verifieras. På detta sätt
```

enligt https://sv.wiktionary.org/wiki/kvitto